service/src/main/java/bio/terra/pipelines/db/entities/PipelineInputsDefinition.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.pipelines.db.entities;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name = "pipeline_inputs_definitions")
```

Comment:
nitpick: should this be `pipeline_input_definitions` ?

Reply:
ugh, yes, it should be. will change